cmake/FindADIOS1.cmake

```diff
@@ -0,0 +1,260 @@
+    foreach(_LIB ${_ADIOS1_LIBS})
+        string(REPLACE " -l" "" _LIB ${_LIB})
+
+        # find static lib: absolute path in -L then default
+        find_library(_LIB_DIR NAMES ${_LIB} PATHS ${ADIOS1_LIBRARY_DIRS} CMAKE_FIND_ROOT_PATH_BOTH)
+
+        # found?
+        if(_LIB_DIR)
+            if(_LIB STREQUAL "adios")
+                message(STATUS "  Found the main adios library in ${_LIB_DIR}")
+                set(ADIOS1_LIBRARY_PATH "${_LIB_DIR}")
+                #message(STATUS "    ADIOS1_LIBRARY_PATH set to ${ADIOS1_LIBRARY_PATH}")
+            else()
+                message(STATUS "  Found ${_LIB} in ${_LIB_DIR}")
+                list(APPEND ADIOS1_DEPENDENCY_LIBRARIES "${_LIB_DIR}")
+            endif()
+        else(_LIB_DIR)
+            set(ADIOS1_FOUND FALSE)
+            message(STATUS "ADIOS1: Could NOT find library '${_LIB}'")
+        endif(_LIB_DIR)
+
+        # clean cached var
+        unset(_LIB_DIR CACHE)
+        unset(_LIB_DIR)
+    endforeach()
+
+    #add libraries which are already using cmake format
+    string(REGEX MATCHALL "/([A-Za-z_0-9/\\.-]+)\\.([a|so]+)" _ADIOS1_LIBS_SUB "${ADIOS1_LINKFLAGS}")
+    list(APPEND ADIOS1_DEPENDENCY_LIBRARIES "${_ADIOS1_LIBS_SUB}")
+
+    # add the version string
+    execute_process(COMMAND ${ADIOS1_CONFIG} -v
+                    OUTPUT_VARIABLE ADIOS1_VERSION
+                    OUTPUT_STRIP_TRAILING_WHITESPACE)
+    
+endif(ADIOS1_FOUND)
+
+# unset checked variables if not found
+if(NOT ADIOS1_FOUND)
+    unset(ADIOS1_INCLUDE_DIRS)
+    unset(ADIOS1_DEPENDENCY_LIBRARIES)
+    unset(ADIOS1_LIBRARY_PATH)
+endif(NOT ADIOS1_FOUND)
+
+
+# restore CMAKE_FIND_LIBRARY_SUFFIXES if manipulated by this module ###########
+if(ADIOS1_USE_STATIC_LIBS)
+    set(CMAKE_FIND_LIBRARY_SUFFIXES ${_ORIG_CMAKE_FIND_LIBRARY_SUFFIXES})
+endif()
+
+
+###############################################################################
+# FindPackage Options
+###############################################################################
+
+# handles the REQUIRED, QUIET and version-related arguments for find_package
+include(FindPackageHandleStandardArgs)
+find_package_handle_standard_args(ADIOS1
+    REQUIRED_VARS ADIOS1_LIBRARY_PATH ADIOS1_DEPENDENCY_LIBRARIES ADIOS1_INCLUDE_DIRS
+    VERSION_VAR ADIOS1_VERSION
+)
+
+##########################################################################
+# Add target and dependencies to ADIOS2
+##########################################################################
+#message(STATUS "ADIOS1 Find ended with ${ADIOS1_FOUND}")
+    if(ADIOS1_FOUND AND NOT TARGET adios1::adios)
+      message(STATUS "Add library ADIOS1 to the build")
+      add_library(adios1::adios UNKNOWN IMPORTED)
+      set_target_properties(adios1::adios PROPERTIES
+        IMPORTED_LOCATION "${ADIOS1_LIBRARY_PATH}"
+        INTERFACE_LINK_LIBRARIES "${ADIOS1_DEPENDENCY_LIBRARIES}"
+        INTERFACE_INCLUDE_DIRECTORIES "${ADIOS1_INCLUDE_DIRS}"
+      )
+    endif()
```

Comment:
Does this also find the no_mpi version of the library?  If it does then it shoudl create another target for it.